common_functions.sh

```diff
@@ -696,3 +696,12 @@ function get_shasums() {
 	chmod +x "${ofile_sums}" "${ofile_build_time}"
 }
 
+# Login to docker for pushing images to repo
+function docker_login() {
+	docker login
```

Comment:
Please add a comment that the credentials are taken from the `.docker/config.json` file. Also it needs to be `docker login docker.io`

Reply:
Thanks for the review @dinogun . Updated the comment and changed `docker login` to `docker login docker.io`